ui/src/utils.tsx

```diff
@@ -221,7 +221,10 @@ export function insertWorkflowToCanvas(json: string, insertPos?: number[]) {
   canvas = null;
 }
 
-export async function generateUniqueName(name?: string) {
+export async function generateUniqueName(
```

Comment:
we shuold move this into workflowsTable later

Reply:
done